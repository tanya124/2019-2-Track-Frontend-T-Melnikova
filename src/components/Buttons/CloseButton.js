import React from 'react';
import styled from '@emotion/styled';
import close from '../../assets/cross.svg';
import PropType from 'prop-types';

const Button = styled.img`
	width: 1.5em;
	margin-right: 10px;
	&:hover {
		filter: invert(1);
	}
`;


function CloseButton(props) {
	return (
		<Button src={ close } onClick={()=>props.handleModal()}/>
	);
}

CloseButton.propTypes = {
	handleModal: PropType.func.isRequired,
};

export default CloseButton;