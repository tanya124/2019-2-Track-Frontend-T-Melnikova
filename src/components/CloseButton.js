import React from 'react';
import styled from '@emotion/styled';
import close from '../assets/cross.svg';

const Button = styled.img`
	width: 1.5em;
	margin-right: 10px;
	&:hover {
		filter: invert(1);
	}
`;


function CloseButton(props) {
	return (
		<Button src={ close } onClick={()=>props.handleModal()}/>
	);
}

export default CloseButton;