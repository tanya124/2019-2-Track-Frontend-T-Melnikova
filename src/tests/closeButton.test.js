import React from 'react';
import renderer from 'react-test-renderer';
import CloseButton from '../components/Buttons/CloseButton';

it('renders correctly', () => {
	const tree = renderer.create(<CloseButton handleModal={jest.fn()} />).toJSON();
	expect(tree).toMatchSnapshot();
});
