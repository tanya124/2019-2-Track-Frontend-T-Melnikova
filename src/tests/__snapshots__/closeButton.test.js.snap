// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<img
  className="css-1vstkwa"
  onClick={[Function]}
  src="cross.svg"
/>
`;
